purchase_request_attachment/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Purchase Request Attachment',
+    'version': '16.0.1.0.0',
+    'summary': """ Purchase Request Attachment """,
+    'author': 'Aginix Technologies',
+    'website': 'https://github.com/aginix/kmitl',
+    'category': 'KMITL',
+    'depends': ['purchase_request_kmitl'],
```

Comment:
depends `purchase_request` แทนได้ไหม

```suggestion
    'depends': ['purchase_request'],
```